explore/src/main/scala/explore/schedulingWindows/SchedulingWindowsTile.scala

```diff
@@ -203,58 +203,57 @@ object SchedulingWindowsTile:
     private val DeleteColId: ColumnId = ColumnId("Delete")
 
     private val component =
-      ScalaFnComponent
-        .withHooks[Props]
-        .useResizeDetector()
-        .useMemoBy((_, _) => ()): (_, _) => // cols
-          _ =>
-            List(
-              ColDef(
-                WindowColId,
-                _._1,
-                size = 400.toPx
-              ).withCell: cell =>
-                <.span(
-                  cell.value.renderVdom,
-                  <.span(Icons.ErrorIcon).withTooltip(BadTimingWindow).unless(cell.value.isValid)
-                ),
-              ColDef(
-                DeleteColId,
-                _._2,
-                size = DeleteColWidth.toPx
-              ).withCell: cell =>
-                Button(
-                  text = true,
-                  onClickE = e =>
-                    e.stopPropagationCB >>
-                      cell.table.options.meta
-                        .map:
-                          _.updateWindows(sws => sws.take(cell.value) ++ sws.drop(cell.value + 1))
-                        .orEmpty
-                ).compact.small(Icons.Trash)
-            )
-        .useMemoBy((props, _, _) => props.windows.get): // rows
-          (_, _, _) => _.zipWithIndex.sorted
-        .useReactTableBy: (props, resize, cols, rows) =>
-          TableOptions(
-            cols,
-            rows,
-            enableRowSelection = true,
-            getRowId = (row, _, _) => RowId(row._2.toString),
-            state = PartialTableState(
-              columnSizing = ColumnSizing(
-                WindowColId -> resize.width.map(w => (w - DeleteColWidth).toPx).getOrElse(400.toPx)
-              ),
-              columnVisibility = ColumnVisibility(
-                DeleteColId -> Visibility.fromVisible(!props.readOnly)
-              )
-            ),
-            meta = TableMeta(props.windows.mod)
-          )
-        .useEffectOnMountBy: (p, _, _, _, table) =>
-          val cb = (a: RowSelection) => table.setRowSelection(a)
-          p.setTileState(TileState(cb))
-        .render: (props, resize, _, _, table) =>
+      ScalaFnComponent[Props]: props =>
+        for {
+          resize <- useResizeDetector
+          // cols
```

Comment:
Probably don't need these comments since the results are now named. But, probably not worth updating the branch to make the changes.